deployment/Makefile

```diff
@@ -1,4 +1,6 @@
-PROJECT_ID := ford3
+export COMPOSE_PROJECT_NAME=ford3
```

Comment:
Make all `docker-compose` command to use `docker-compose -p ford3` automatically.